connectors/agentic-ai/src/main/java/io/camunda/connector/agenticai/a2a/client/api/A2aMessageSender.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.agenticai.a2a.client.api;
+
+import io.a2a.spec.AgentCard;
+import io.camunda.connector.agenticai.a2a.client.model.A2aOperationConfiguration.SendMessageOperationConfiguration;
+import io.camunda.connector.agenticai.a2a.client.model.result.A2aSendMessageResult;
+
+public interface A2aMessageSender {
+  A2aSendMessageResult sendMessage(
+      SendMessageOperationConfiguration sendMessageOperation, AgentCard agentCard);
```

Comment:
💭 Nit: from a logical/hierarchical perspective, would make `(agentCard, sendMessageOperation)` more sense (thinking of I can send multiple message to a single agent card)? This would also align with `(agentCard, consumer)` in the client factory.